tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -276,7 +276,7 @@
             edpm_nodes_validation_validate_controllers_icmp: false
             edpm_nodes_validation_validate_gateway_icmp: false
 
-            edpm_chrony_ntp_servers: {{ edpm_chrony_ntp_servers | default(default_edpm_chrony_ntp_servers) }}
```

Comment:
we'll need to update that as well https://github.com/rdo-infra/rdo-jobs/blob/c18955ec6951e50a5cf75b6662c9ab015f42d03c/playbooks/data_plane_adoption/deploy_standalone_run_repo_tests.yaml#L186

maybe we just set both until this merges?

Reply:
ok found it you already posted it https://review.rdoproject.org/r/c/rdo-jobs/+/50837

would be nice to reference that in the commit message

Reply:
ok marking this resolved as we  merged /r/c/rdo-jobs/+/50837